### DISCUSSION TITLE
Cheats?

### DISCUSSION THREAD

**Discussion by Hislasthope** [nzp-team/nzportable#740] [General]
is there any way to make a command cheat like a pack a punch

**Comment by Useless3** [comment]
If you haven't found anything, here's a cheat: `/give <weapon ID>` using /give and entering a specific weapon ID, will give you that weapon. For example, to get a raygun, you can use `/give 20` and you get the raygun. It goes through 1-59. I found a list on the official NZP docs with weapon IDs, which includes every weapon
 https://docs.nzp.gay/internal/weapon-ids
The IDs I know so far are:
- Mustang and Sally - 28
- The Reaper - 48
- Wunderwaffle - 56
- Wunderwaffle DG-2 - 57
- Porter's X2 Ray-Gun - 33
- PPSh-41 - 18

**Reply by MotoLegacy** [reply]
> but it doesn't include EVERY weapon (I think I haven't checked in a while);

Of course it does, it was complete when the page was first uploaded

**Reply by katniny** [reply]
The docs do have every weapon, including their pack-a-punched variants.

**Reply by Useless3** [reply]
Oh ok thanks
